lumigator/python/mzai/sdk/tests/data/dialogsum_exc.csv

```diff
@@ -0,0 +1,55 @@
+examples,ground_truth
```

Comment:
Since both SDK and BE tests are now using this, how about placing it in an upper level, like `python/mzai/sample_data`? (Not sold on the name or location, be my guest 😁 )

Reply:
Sure! I can move it on #290, for example. The sooner we get some of these tests in, the better :)

Thank you @javiermtorres ! 🙌